index.html

```diff
@@ -13,7 +13,24 @@
 
 <body>
     <div class="container">
-   
+
+
+      <div class="topnav">
+        <div class="lambda-black">
+          <img src="img/lambda-black.png" alt="">
+        </div>
+        <nav>
+          <a href="Sprint-Challenge--User-Interface\index.html">Home</a>
+          <a href="Sprint-Challenge--User-Interface\about.html">About</a>
+          <a href="#">Products</a>
+          <a href="">Blog</a>
+          <a href="#">Contact</a>
+        </nav>
+
+      </div>
+    <div class="jumbopic">
```

Comment:
For readability, the div element here (class="jumbopic") should be indented to the same level as the div of class = "topnav". A tool like the the IDE extension prettier helps with this.